### ISSUE TITLE
Problem in delete mailbox

### ISSUE THREAD

**Issue by tintusumith** [barbushin/php-imap#226]
1.Hi, When I am trying to delete a mailbox that created manually in my account(ex: test), some error showing. I think its due to one extra '.' appended between mailbox path and mailbox name.
Please refer the code given below.
public function deleteMailbox($name) {
$this->imap('deletemailbox', $this->imapPath . '.' . $name);
}
2.When I create a mailbox using createMailbox($name), one extra '.' appended before the mailbox name. This issue exists in rename function also

**Comment by aymanrb** [comment]
Should be related to Issue #213 -  

Might be partially fixed by #264 - allowing you to set the path delimiter to it's correct value instead of always using the dot **.**

**Comment by Sebi94nbg** [comment]
Commit [ad2334e](https://github.com/barbushin/php-imap/commit/ad2334e579d82a83d244009cf84f57a8c2038ffc) has solved this issue.

Just tested those cases again and all worked fine:

1. Create new mailbox folder using `php-imap`
2. Rename before created folder with `php-imap`
3. Delete before renamed folder with `php-imap`
4. Create new mailbox folder using normal client (eg. Outlook OWA)
5. Rename before created folder with `php-imap`
6. Delete before renamed folder with `php-imap`

The mentioned fix is already included in the current stable version 3.0.10.

```php
// Change path delimiter from default '.' to '/'
$mailbox->setPathDelimiter('/');

// Create new mailbox folder 'test'
$mailbox->createMailbox('test');

// Rename mailbox folder 'test' to 'old'
$mailbox->renameMailbox('test', 'old');

// Delete mailbox folder 'old'
$mailbox->deleteMailbox('old');
```
